src/m365/flow/commands/environment/environment-list.ts

```diff
@@ -25,10 +25,12 @@ class FlowEnvironmentListCommand extends AzmgmtCommand {
     try {
       const res = await odata.getAllItems<{ name: string, displayName: string; properties: { displayName: string } }>(`${this.resource}providers/Microsoft.ProcessSimple/environments?api-version=2016-11-01`);
 
-      if (args.options.output !== 'json' && res.length > 0) {
-        res.forEach(e => {
-          e.displayName = e.properties.displayName;
-        });
+      if (res.length > 0) {
+        if (args.options.output !== 'json') {
+          res.forEach(e => {
+            e.displayName = e.properties.displayName;
+          });
+        }
 
         await logger.log(res);
```

Comment:
Shouldn't we move this logger one line down out of the `if (res.length > 0)` condition? In the very unlikely event that there are no results returned, the command doesn't output anything at all. When nothing was found, list commands output an empty array instead of nothing.

Reply:
That would change the behavior of the command which could be considered breaking. I didn't want to include that in a minor update.

Reply:
Hmm, ok yeah fair point, let's create an issue for the v8 milestone :)